src/pages/UserAccount/__tests__/__snapshots__/index.spec.tsx.snap

```diff
@@ -0,0 +1,340 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`UserAccount page should correctly render the component which contains profile data 1`] = `
+Array [
+  <section
+    className="pf-c-page__main-section pf-m-light"
+    style={
+      Object {
+        "paddingBottom": "0",
+      }
+    }
+  >
+    <h1
+      className="pf-c-title pf-m-2xl"
+    >
+      Account
+    </h1>
+    <div
+      className="pf-c-content"
+      style={
+        Object {
+          "paddingTop": "10px",
+        }
+      }
+    >
+      <p
+        className=""
+        data-pf-content={true}
+      >
+        This is where you can view and edit your account information for Eclipse Che.
```

Comment:
we must use branding instead of Eclipse Che here

Reply:
I have fixed it